### PR TITLE
Ocd 2474 - update invalid input fields

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: PENDING_RELEASE_NOTES.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ _Date TBD_
   * Reason for status change
   * Reason for listing change
 * Prevent users from using the following macra measures which are under review: RT13 EH/CAH Stage 3, RT14 EH/CAH Stage 3, RT15 EH/CAH Stage 3 
+* Update invalid error and warning messages for upload file
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]: STG is frozen, so these changes need to be in a PENDING_PENDING file

Comment (kekey1) [inline PENDING_RELEASE_NOTES.md]: Should be in PENDING_PENDING I think?

File: chpl/chpl-resources/src/main/resources/errors.properties [hunk 2]
```diff
@@ -371,7 +371,7 @@ user.subjectName.required=Username ('subject name') is required.
 user.providerKey.invalid=Provided user key is not valid in the database. The user key is valid for up to 3 days from when it is assigned.
 
 #Warnings
-transparencyAttestation.save=Warning: The transparency attestation for the developer is different in the system than in the upload file. To save the updated transparency attestation, as well as all other developer information, please click 'Save as Developer Information' above before proceeding.
+transparencyAttestation.save=The transparency attestation for the developer is different in the system than in the upload file. To save the updated transparency attestation, as well as all other developer information, please click 'Save as Developer Information' above before proceeding.
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/errors.properties]: I can see one change to a message, but I count three in the Excel file. Are the other two not actually supposed to change?

Comment (brianlindsy) [reply chpl/chpl-resources/src/main/resources/errors.properties]: Those two other errors already have the correct text in the code

**Comment by brianlindsy** [comment]
Looks like all tests pass in Jenkins now

**Pull request merged**